### DISCUSSION TITLE
Select color of rooms

### DISCUSSION THREAD

**Discussion by AseKarlsson** [humbertogontijo/homeassistant-roborock#359] [General]
Hi,
Ended up here after the Xiaomi Miio integration failed using Piotrs great integration using map_extractor.
Almost everything is here, but can I somehow change colors of the rooms?

**Comment by humbertogontijo** [comment]
Currently it is not possible to change the colors

**Reply by AseKarlsson** [reply]
@humbertogontijo, Thanks for this integration and thanks for a straight reply! Is this something coming?

**Comment by Mar1usW3** [comment]
any update here? I have 2 times 2 rooms directly next to each other with the same color. ![image](https://github.com/humbertogontijo/homeassistant-roborock/assets/66533871/cf04e49f-113a-44e4-8242-d4c9e4ec4efa)
